mdoc/Test/en.expected-javascript/Mono.DocTest.Generic/GenericBase`1.xml

```diff
@@ -0,0 +1,149 @@
+      <ReturnValue>
+        <ReturnType>System.Int32</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="ItemChanged">
+      <MemberSignature Language="C#" Value="public event Action&lt;Mono.DocTest.Generic.MyList&lt;U&gt;,Mono.DocTest.Generic.MyList&lt;U&gt;.Helper&lt;U,U&gt;&gt; ItemChanged;" />
+      <MemberSignature Language="ILAsm" Value=".event class System.Action`2&lt;class Mono.DocTest.Generic.MyList`1&lt;!U&gt;, class Mono.DocTest.Generic.MyList`1/Helper`2&lt;!U, !U, !U&gt;&gt; ItemChanged" />
+      <MemberSignature Language="JavaScript" Usage="function onItemChanged(eventArgs) { /* Your code */ }&#xA;genericBase.addEventListener(&quot;itemchanged&quot;, onItemChanged);&#xA;genericBase.removeEventListener(&quot;itemchanged&quot;, onItemChanged);&#xA;- or -&#xA;genericBase.onitemchanged = onItemChanged;" />
+      <MemberType>Event</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Action&lt;Mono.DocTest.Generic.MyList&lt;U&gt;,Mono.DocTest.Generic.MyList&lt;U&gt;+Helper&lt;U,U&gt;&gt;</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="MyEvent">
+      <MemberSignature Language="C#" Value="public event EventHandler&lt;Mono.DocTest.Generic.GenericBase&lt;U&gt;.FooEventArgs&gt; MyEvent;" />
+      <MemberSignature Language="ILAsm" Value=".event class System.EventHandler`1&lt;class Mono.DocTest.Generic.GenericBase`1/FooEventArgs&lt;!U&gt;&gt; MyEvent" />
+      <MemberSignature Language="JavaScript" Usage="function onMyEvent(eventArgs) { /* Your code */ }&#xA;genericBase.addEventListener(&quot;myevent&quot;, onMyEvent);&#xA;genericBase.removeEventListener(&quot;myevent&quot;, onMyEvent);&#xA;- or -&#xA;genericBase.onmyevent = onMyEvent;" />
+      <MemberType>Event</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler&lt;Mono.DocTest.Generic.GenericBase&lt;U&gt;+FooEventArgs&gt;</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="op_Explicit">
+      <MemberSignature Language="C#" Value="public static explicit operator U (Mono.DocTest.Generic.GenericBase&lt;U&gt; list);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig specialname !U op_Explicit(class Mono.DocTest.Generic.GenericBase`1&lt;!U&gt; list) cil managed" />
+      <MemberSignature Language="JavaScript" Value="function op_Explicit(list)" Usage="var u = Mono.DocTest.Generic.GenericBase.op_Explicit(list)" />
```

Comment:
I was under the impression that conversion operators weren't supported in js ? It could very well be that `op_Explicit` can be called, but can you verify that this is in fact supported? is it mentioned anywhere in the documentation?

Reply:
Yes, operators overloading is not supported by WinRT. I was a little confused by this phrase in e-mail 
>C# implementations get compiled to metadata as static methods

But it can't be compiled in WinRT project: 
`'SampleComponent.Example.op_Addition(SampleComponent.Example, SampleComponent.Example)' is an operator overload. Managed types cannot expose operator overloads in the Windows Runtime`
So, I've added this check to `IsMethodSupported` method:
```
!DocUtils.IsOperator(methodDefinition)
```